### ISSUE TITLE
Carousel - Infinite circular scrolling if having less items than the numScroll

### ISSUE THREAD

**Issue by KyubiBoy** [primefaces/primeng#8601]
[ x ] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

**Current behavior**
Carousel settings: numScroll=5, numVisible=5, circulat=true
if I have 5 items in carousel, everything works fine, but if my list has 4 or less items, the carousel will infinitely scroll in either way.

**Expected behavior**
It should not scroll at all or it should loop like it does when you have the exact number of items as set in numScroll and numVisible.

* **Angular version:** 8.0.0

* **PrimeNG version:** 8.1.1

* **Browser:** Chrome 
 
* **Language:** TypeScript

**Comment by Rotzbua** [comment]
fixed in prime v9 and tls v8 by #8500

**Comment by ratsey** [comment]
Broken in 17.18.11 as well as 18 Betas revisions 2 and 3. This can be easily reproduced using the example Stackblitz in the docs by reducing the sample data to 3 products and changing the responsive settings to show 3. Then adjust the window width as needed.

Example: [Stackblitz](https://c41uvf.stackblitz.io)
